Makefile

```diff
@@ -4,7 +4,7 @@ CONTAINER_OPTS ?= --security-opt $(shell grep -q selinux /sys/kernel/security/ls
 CONTAINER_BUILD_OPTS ?=
 CONTAINER_CONFIG_DIR ?= testing/containers/ceph
 VOLUME_FLAGS ?=
-CEPH_VERSION ?= pacific
+CEPH_VERSION ?= nautilus
```

Comment:
Please don't change this.  @anoopcs9 recently changed it to ?= so that you should be able to simply set CEPH_VERSION in your environment for convenience.

Reply:
My bad - didn't mean to commit that!